tests/components/comelit/conftest.py

```diff
@@ -82,10 +84,10 @@ def mock_vedo() -> Generator[AsyncMock]:
         ),
     ):
         vedo = mock_comelit_vedo.return_value
-        vedo.get_all_areas_and_zones.return_value = VEDO_DEVICE_QUERY
+        vedo.get_all_areas_and_zones.return_value = deepcopy(VEDO_DEVICE_QUERY)
```

Comment:
same about `deepcopy`

Reply:
Without the tests that are parametrized don't work as they get previous data at the next run.